### ISSUE TITLE
Peroneal tendon injury

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11168]
**Preferred term label:**
Peroneal tendon injury

**Synonyms**

**Definition (free text, please give PubMed ID)**
Ankle MRI or ultrasound showing edema or thickening of peroneal tendon(s), fluid within peroneal tendon sheath, partial or complete tear in one or both of the peroneal tendons, or subluxation/dislocation of one or both of the tendons

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Peroneal tendon disorder: https://www.ncbi.nlm.nih.gov/books/NBK544354/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001409
- label: Peroneal tendon injury
- parents: [Abnormal ankle morphology (HP:0034673)](https://hpo.jax.org/app/browse/term/HP:0034673)
- pmid: PMID:31335074
- def: Ankle MRI or ultrasound showing edema or thickening of peroneal tendon(s), fluid within peroneal tendon sheath, partial or complete tear in one or both of the peroneal tendons, or subluxation/dislocation of one or both of the tendons.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11168
- synonyms:

**Comment by MickeySegal** [comment]
In entering this and HP:6001408 (github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11167) into our system I see that both included subluxation.  To make them more distinct I removed that, changing our definition to be "Ankle MRI or ultrasound showing edema or thickening of peroneal tendon(s), fluid within peroneal tendon sheath, partial or complete tear in one or both of the peroneal tendons."

**Comment by pnrobinson** [comment]
@MickeySegal  thanks!
I agree with this and have updated the definition accordingly 
Ankle MRI or ultrasound showing edema or thickening of peroneal tendon(s), fluid within peroneal tendon sheath, or a partial or complete tear in one or both of the peroneal tendons.
